drivers/sensor/bmp388/bmp388.c

```diff
@@ -240,6 +336,14 @@ static int bmp388_attr_set_odr(const struct device *dev,
 		return odr;
 	}
 
+	#if defined (CONFIG_BMP388_VALIDATE_TIMINGS)
+	/* check that the new ODR would be consistent with the current OSRs */
+	if ( 0 > bmp388_validate_osr_and_odr_settings(odr, data->osr_pressure, data->osr_temp))
```

Comment:
```suggestion
	if (bmp388_validate_osr_and_odr_settings(odr, data->osr_pressure, data->osr_temp) < 0)
```
Please do this throughout